recipes/pyophidia/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: pyophidia
+    version: "1.4.0"
+
+source:
+    url: git://github.com/OphidiaBigData/PyOphidia/archives/v1.4.0.tar.gz
+    sha256: b276b11d8e72396f00a95eeec0086df75e6fe6980106eb2fab68d237c665a1f7
+
+build:
```

Comment:
It looks like this package might be a candidate for using `noarch`:

https://conda-forge.org/docs/meta.html#building-noarch-packages

Reply:
I though conda-forge couldn't do `noarch`, i'll look into this.